### ISSUE TITLE
start  download time

### ISSUE THREAD

**Issue by ali-libre** [PaperAirplane-Dev-Team/GigaGet#3]
Hi
Pls,add start time for download
best  regards

**Comment by Harry-Chen** [comment]
Do you mean the feature of not starting download immediately?

**Comment by ali-libre** [comment]
No.schedule. In example an option  do determind some download start at 2.am .
